docs/ui/AppearanceDemo.tsx

```diff
@@ -48,6 +49,18 @@ export const AppearanceDemo = ({
       children
     );
 
+  let messageTitle = '';
```

Comment:
I think this is a bit messy with all the `else if`. Also, `isVariantOrSizeMissing` can be inferred by just checking the `messageTitle` (why is it a title btw 😜).

Alternatively: Is it enough to just put a more generic message here? Like "Appearance options are not available with the current theme."